app/routers/event_images.py

```diff
@@ -0,0 +1,216 @@
+    'halloween': 'halloween',
+    'helloween': 'halloween',
+    "hallowe'en": 'halloween',
+    'allhalloween': 'halloween',
+    "all hallows' eve": 'halloween',
+    "all saints' Eve": 'halloween',
+    'hiking': 'hike',
+    'hike': 'hike',
+    'hikes': 'hike',
+    'kayaking': 'kayak',
+    'piano': 'music',
+    'singing': 'music',
+    'music class': 'music',
+    'choir practice': 'music',
+    'flute': 'music',
+    'orchestra': 'music',
+    'oboe': 'music',
+    'clarinet': 'music',
+    'saxophone': 'music',
+    'cornett': 'music',
+    'trumpet': 'music',
+    'contrabass': 'music',
+    'cello': 'music',
+    'trombone': 'music',
+    'tuba': 'music',
+    'music ensemble': 'music',
+    'string quartett': 'music',
+    'guitar lesson': 'music',
+    'classical music': 'music',
+    'choir': 'music',
+    'manicure': 'manicure',
+    'pedicure': 'manicure',
+    'manicures': 'manicure',
+    'pedicures': 'manicure',
+    'massage': 'massage',
+    'back rub': 'massage',
+    'backrub': 'massage',
+    'massages': 'massage',
+    'pills': 'pill',
+    'medicines': 'pill',
+    'medicine': 'pill',
+    'drug': 'pill',
+    'drugs': 'pill',
+    'ping pong': 'pingpong',
+    'table tennis': 'pingpong',
+    'ping-pong': 'pingpong',
+    'pingpong': 'pingpong',
+    'plan week': 'plan',
+    'plan quarter': 'plan',
+    'plan day': 'plan',
+    'plan vacation': 'plan',
+    'week planning': 'plan',
+    'vacation planning': 'plan',
+    'pokemon': 'pokemon',
+    'reading': 'read',
+    'newspaper': 'read',
+    'fridge repair': 'repair',
+    'handyman': 'repair',
+    'electrician': 'repair',
+    'diy': 'repair',
+    'jog': 'ran',
+    'jogging': 'ran',
+    'running': 'ran',
+    'jogs': 'ran',
+    'runs': 'ran',
+    'sail': 'sail',
+    'sailing': 'sail',
+    'boat cruise': 'sail',
+    'sailboat': 'sail',
+    'santa claus': 'santa',
+    'father christmas': 'santa',
+    'skiing': 'ski',
+    'ski': 'ski',
+    'skis': 'ski',
+    'snowboarding': 'ski',
+    'snowshoeing': 'ski',
+    'snow shoe': 'ski',
+    'snow boarding': 'ski',
+    'soccer': 'soccer',
+    'swim': 'swam',
+    'swimming': 'swam',
+    'swims': 'swam',
+    'tennis': 'tennis',
+    'thanksgiving': 'thanksgiving',
+    'wedding': 'wed',
+    'wedding eve': 'wed',
+    'wedding-eve party': 'wed',
+    'weddings': 'wed',
+    'christmas': 'christmas',
+    'xmas': 'christmas',
+    'x-mas': 'christmas',
+    'yoga': 'yoga',
+}
+
+
+def generate_flare_link_from_lemmatized_word(lemmatized_word: str) -> str:
+    """Generate a link to a flair by a given lemmatized word.
+
+    Args:
+        lemmatized_word (str): The lemmatized word.
+
+    Returns:
+        str: The suitable link.
+    """
+    return f'{FLAIRS_REL_PATH}\\{lemmatized_word}{FLAIRS_EXTENSION}'
+
+
+def remove_non_alphabet_chars(s: str) -> str:
+    """Remove non-alphabet chars from a given string
+
+    Args:
+        s (str): The string to remove the non-alphabet chars from.
```

Comment:
Would be better to change the name to something recognizable like text, word etc'